examples/atmajs/readme.md

```diff
@@ -80,28 +86,29 @@ Some key points of the library are:
 
 Pages benefits from component based approach. Each component's controller can define caching settings, so that the component renders only once. Developer can restrict any component for the server side rendering only, so that the controller itself and any sensitive data is not sent to the client. When a component is rendered, then only HTML is sent to the client, _where all bindings and event listeners are attached_. So it is extremely SEO friendly.
 
-
-
 Here are some links you may find helpful:
 
 - [Get Started](http://atmajs.com/get/github)
 - [Mask Markup Live Test](http://atmajs.com/mask-try)
+- Mask Syntax Plugins
+	- [Sublime](https://github.com/tenbits/sublime-mask)
+	- [Atom](https://github.com/tenbits/package-atom)
```

Comment:
As for now, no more other plugins exist. For WebStorm and VisualStudio are on the way.